relevanceai/operations_new/sentiment/base.py

```diff
@@ -103,8 +108,19 @@ def analyze_sentiment(
         sentiment = labels[0][ind_max]["label"]
         max_score = labels[0][ind_max]["score"]
         sentiment = self.label_mapping.get(sentiment, sentiment)
-        if sentiment.lower() == "neutral":
+        if sentiment.lower() == "neutral" and max_score > self.sensitivity:
             overall_sentiment = 1e-5
```

Comment:
Might be worth considering the option to have neutral sentiment as 0.2, as some neutral sentiment are more on the positive side